src/main.rs

```diff
@@ -72,12 +72,11 @@ enum Backend {
 #[derive(StructOpt)]
 struct GlobalArgs {
     /// Host or address to listen on.
-    #[structopt(
-        long = "host",
-        default_value = "0.0.0.0:8080",
-        env = "RUDOLFS_HOST"
-    )]
-    host: String,
+    #[structopt(long = "host", env = "RUDOLFS_HOST")]
+    host: Option<String>,
```

Comment:
I had to think about this for a little while, but I don't think this is a breaking change since a port can still be specified with `--host foobar:8080`. :+1:

Before, when `default_value` was used, `StructOpt` would generate some info about the default value. Since that's now gone, can the doc comment be expanded to something like this?
```rust
/// The host or address to listen on. If this is not specified, then `0.0.0.0`
/// is used where the port can be specified with `--port` (port 8080 is used by
/// default if that is also not specified).
```

Reply:
Sure I will add this in.